lms/services/canvas_api/_authenticated.py

```diff
@@ -35,16 +35,31 @@ def __init__(  # pylint: disable=too-many-arguments
         self._client_secret = client_secret
         self._redirect_uri = redirect_uri
 
-    def send(self, method, path, schema, params=None):
+    def send(
+        self, method, path, schema, params=None, timeout=(10, 10)
+    ):  # pylint:disable=too-many-arguments
```

Comment:
I'm not sure that we can avoid the `# pylint:disable=too-many-arguments` here without spending more time refactoring this code than I want to right now. (`HTTPService` avoids this problem by using `**kwargs` to allow arbitary kwargs to be passed through to requests. But it generally has a much "thinner" design where requests is wrapped in fewer layers.)

Reply:
Note that I added the `timeout` argument _before_ the `params` argument which changes the order of the arguments and could possibly break some callers. The only thing that calls this is `CanvasAPIClient` (in `client.py`) and it always passes `params` as a keyword argument, so nothing is broken